src/SimpleAuth/Providers/DropBox.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SimpleAuth.Providers
+{
+    public class DropBoxAuth : OAuthApi
+    {
+        public DropBoxAuth(string client_id, string client_secret, HttpMessageHandler handler = null)
+            : base("DropBox", client_id, client_secret, handler)
+        {
+            this.Scopes = new[] { "WTF" };
+            this.TokenUrl = "https://api.dropbox.com/1/oauth2/token";
+        }
+
+        protected override WebAuthenticator CreateAuthenticator()
+        {
+            return new DropBoxAuthenticator
+            {
+                ClientId = ClientId,
+            };
+        }
+
+        protected override Task<OAuthAccount> GetAccountFromAuthCode(WebAuthenticator authenticator, string identifier)
+        {
+            var author = (DropBoxAuthenticator)authenticator;
+            var account = new OAuthAccount()
+            {
+                ExpiresIn = 3600, // ??
```

Comment:
If they never expire, set it to -1. If not it will request tokens regularly. And then Refresh token is not required.